### ISSUE TITLE
Paste images from clipboard 

### ISSUE THREAD

**Issue by Fonsos** [barredo/mastodeck#61]
While creating a post, it would be awesome if you could paste from the clipboard

**Comment by VK2AAF** [comment]
Yes, please. Eliminates the step of saving a screencap as a file before posting to Mastodeck. 

Mastodeck keeps getting better & better, it's truly the BTD of Mastodon, thanks Alex & Co :)
